Mathlib/GroupTheory/SpecificGroups/Alternating.lean

```diff
@@ -259,7 +258,6 @@ theorem normalClosure_finRotate_five : normalClosure ({⟨finRotate 5,
           normalClosure _ :=
         SetLike.mem_coe.1 (subset_normalClosure (Set.mem_singleton _))
       exact (mul_mem (Subgroup.normalClosure_normal.conj_mem _ h
-        -- Porting note: added `: _`
         ⟨Fin.cycleRange 2, Fin.isThreeCycle_cycleRange_two.mem_alternatingGroup⟩) (inv_mem h) :))
```

Comment:
Do we still need the `:` here? If so, it seems to me like the porting note still applies.

Reply:
You are right.